style.css

```diff
@@ -432,3 +434,280 @@ main {
   padding: 0;
   padding-top: 10px;
 }
+
+.top-nav {
+  display: none;
+}
+
+.picture {
+  display: none;
+}
+
+.right-design-desktop {
+  display: none;
+}
+
+.left-design-desktop {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .desktop-introduction .desktop-green-h1 {
+    display: none;
+  }
+
+  .main-section {
+    background-image: url(desktop-images/main.svg);
+    background-size: cover;
+    background-position: center;
+    background-clip: border-box;
+    background-repeat: no-repeat;
+    padding: 60px 0;
+    justify-content: center;
+  }
+
+  .social-media {
+    margin-top: 6rem;
+  }
+
+  .top-nav {
+    display: flex;
+    list-style: none;
+    gap: 10px;
+    text-align: center;
+    margin: auto;
+    padding-bottom: 50px;
+    justify-content: center;
+    color: #172b4d;
+  }
+
+  nav {
+    margin-top: 0;
+  }
+
+  .my-recent-works {
+    display: flex;
+    text-align: center;
+    margin-bottom: 2rem;
+    justify-content: center;
+  }
+
+  .break {
+    display: none;
+  }
+
+  .d-first-paragraph {
+    line-height: 50px;
+  }
+
+  .down-button {
+    margin-top: 0;
+    margin-bottom: 100px;
+  }
+
+  nav .hamburgers {
+    display: none;
+  }
+
+  .commence {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 300px;
+  }
+
+  .first-paragraph {
+    height: 350px;
+    line-height: 32px;
+    width: 700px;
+    align-items: center;
+    text-align: center;
+    padding: 0;
```

Comment:
- [ ] I would suggest you to use a proper css rule  on your width kindly use percentage instead of pix this will help your UI to be more responsive on all devices